lib/network/modules/components/edges/util/EdgeBase.js

```diff
@@ -491,8 +495,13 @@ class EdgeBase {
     ctx.fillStyle = ctx.strokeStyle;
     ctx.lineWidth = this.getLineWidth(selected, hover);
 
-    // draw arrow at the end of the line
-    ctx.arrow(arrowData.point.x, arrowData.point.y, arrowData.angle, arrowData.length);
+    if (arrowData.type && arrowData.type.toLowerCase() === 'circle') {
```

Comment:
Suggestion: Should we better have something like an enum type holding the different edge types (circle, arrow, ...)? Then the types would be stored in one place rather than distributed through the whole code as strings. However, for now this isn't a problem.